src/__tests__/Header.test.js

```diff
@@ -0,0 +1,8 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import Header from '../components/Header';
+
+it('testing rendering Header component', () => {
+  const tree = renderer.create(<Router><Header /></Router>).toJSON;
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] <b>REQUIRED:</b> You only tested Snapshot. As the project requires, kindly add some unit and integration tests as well.
![Screen Shot 2022-03-13 at 04 35 40](https://user-images.githubusercontent.com/45217364/158042544-798fcf2a-319a-4b20-b0d5-83d5acb38676.png)